src/reducers/LoginReducer.js

```diff
@@ -12,9 +13,15 @@ const loginReducer = (state = initialState, action) => {
         ...state,
         isLoading: true,
       };
+    case actionTypes.LOGIN_ERROR:
+      return {
+        ...state,
+        isLoading: false,
```

Comment:
Why aren't we setting the state to the initial state here? I feel its better clarity wise. Besides that i have no further comments